src/meta/app/src/schema/index.rs

```diff
@@ -176,6 +178,15 @@ pub struct GetIndexReply {
     pub index_id: u64,
     pub index_meta: IndexMeta,
 }
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, PartialEq, Eq)]
+pub struct UpdateIndexReq {
+    pub index_id: u64,
+    pub index_name: String,
+    pub index_meta: IndexMeta,
+}
+
```

Comment:
It looks `index_name` is not used at all when updating index.

Reply:
`index_name` is used in the error message.